css/style.css

```diff
@@ -0,0 +1,66 @@
+body {
+    background-color: #39393A;
```

Comment:
You might be interested in investigating [CSS custom properties](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties), also known as CSS variables. They work just like JS variables, allowing you to cache the same style in one place in case you want to make quick changes in the future without having to find every instance of a particular value. I notice this color is used in a couple places.

You don't have enough repeated values to really demand the use of custom properties here, but it might be something you're interested in in the future.